src/shared/theme.js

```diff
@@ -24,9 +24,9 @@ export default {
   },
   post: {
     fontSizes: {
-      xxl: '2.25rem',
+      xxl: '2.5rem',
       xl: '1.75rem',
-      lg: '1.5rem',
+      lg: '1.25rem',
```

Comment:
put these back to original values. This object post.fontSizes is for Blog Posts only. You will need to add these values to the fontSizes under global.fontsizes above in this file.